.github/workflows/check-authkit.yml

```diff
@@ -0,0 +1,35 @@
+name: Authkit & Login-Handler check
```

Comment:
Something something linting?

Reply:
Ye, thought about that too already. I just tried to `extends` the `frontend` config, but that doesn't work because that uses plenty of plugins. We planned to consolidate all eslint configs of multiple of our projects into a shared one. So I would add linting here as part of doing that.